src/spikeinterface/postprocessing/spike_locations.py

```diff
@@ -92,6 +92,17 @@ def _select_extension_data(self, unit_ids):
         new_spike_locations = self.data["spike_locations"][spike_mask]
         return dict(spike_locations=new_spike_locations)
 
+    def _merge_extension_data(
+        self, units_to_merge, new_unit_ids, new_sorting_analyzer, keep_mask=None, verbose=False, **job_kwargs
+    ):
+        new_spike_locations = self.data["spike_locations"]
+        if keep_mask is not None:
+            new_spike_locations = new_spike_locations[keep_mask]
+
+        ### In theory here, we should recompute the locations since the peak positions
+        ### in a merged could be different. Should be discussed
```

Comment:
I agree but we were OK that mode=soft merge is not 100% extact.
But this should detail in the doc somewhere. explicitly.